### DISCUSSION TITLE
Disable copy/paste files via clipboard

### DISCUSSION THREAD

**Discussion by Sparkblender** [Ylianst/MeshCentral#4166] [Q&A]
I have a use case, where we need to disable copying/pasting files via the clipboard.  Ideally, we would like to grant permission to the user to only copy text, but disallow user from copying/pasting files from/to the remote device.
Is this possible?
Thanks.

**Comment by folosleg** [comment]
Is it possible to copy/paste files through the clipboard at all? How? WebUI or Router? I can't do that at all... Cheers
